engine/client/cl_scrn.c

```diff
@@ -101,6 +101,18 @@ void SCR_DrawFPS( void )
 		/*if( !avgrate ) avgrate = ( maxfps - minfps ) / 2.0f;
 		else */avgrate += ( calc - avgrate ) / host.framecount;
 
+		int strobeInterval = r_strobe->integer;
+		int eFPS; //Effective FPS (strobing effect)
```

Comment:
Move variable defintions to the start of the block for C89(because MSVC) compliance.